### PR TITLE
Add dev-notes/metadata.md

### PR DESCRIPTION
**Pull request by arturkow2000**

### CODE REVIEW

File: dev-notes/metadata.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Platform metadata
+
+Platform metadata contains information that uniquely identify the platform being
+attested. Metadata is composed of
+
+- Device manufacturer and model name
+- MAC address
+- Serial number
+- EK certificate hash
```
Comment (TomaszAIR) [inline dev-notes/metadata.md:9]: @arturkow2000 shouldn't this be removed? The EK cert hash in not a part of metadata?

File: dev-notes/metadata.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Device manufacturer and model name
+
+Human-readable device name as provided by firmware. On x86 this information should be fetched from SMBIOS `System Information/Manufacturer` and
```
Comment (TomaszAIR) [inline dev-notes/metadata.md:13]: @arturkow2000 please wrap at 80

File: dev-notes/metadata.md (hunk 1 above)
Comment (arturkow2000) [reply dev-notes/metadata.md]: Yes, it should.
Fixed.

File: dev-notes/metadata.md (hunk 2 above)
Comment (arturkow2000) [reply dev-notes/metadata.md]: Fixed

**Pull request merged**